### ISSUE TITLE
(Bug) Crash with stacked windows

### ISSUE THREAD

**Issue by hardBSDk** [pop-os/cosmic-comp#617]
When you stack windows the "Float window" is enabled, if I disable it the compositor crash.

- Does it happen because some programs require a floating window and their geometry is hardcoded?

**Comment by Drakulix** [comment]
Works fine on my side, which version of cosmic-comp are you testing this with?

**Comment by hardBSDk** [comment]
@Drakulix The COSMIC packages from the Fedora COPR repository on the cosmic-epoch README.

**Comment by Drakulix** [comment]
> @Drakulix The COSMIC packages from the Fedora COPR repository on the cosmic-epoch README.

Yeah, those are not build by us and that information doesn't tell me a lot. Can you please query `dnf` for the exact version of the package you have installed?

**Comment by hardBSDk** [comment]
@Drakulix Sorry, I was using a build from 4 days ago:

https://copr.fedorainfracloud.org/coprs/ryanabx/cosmic-epoch/build/7730157/

Now I updated to this build:

https://copr.fedorainfracloud.org/coprs/ryanabx/cosmic-epoch/build/7740371/

**Comment by hardBSDk** [comment]
Fixed.
